src/sentry/db/models/fields/bounded.py

```diff
@@ -66,7 +66,7 @@ class BoundedBigAutoField(models.AutoField):  # type: ignore
         MAX_VALUE = 9223372036854775807
 
         def db_type(self, connection: BaseDatabaseWrapper) -> str:
-            return "bigserial"
+            return "int64"
```

Comment:
What is this for ?

Reply:
this needs to be fixed. somehow this type makes it into the generated google standard sql statement without any sort of translation. So we need to either make `db_type` check the database backend somehow, or stop using our own bigautofield.